packages/frontend/src/routes/app/reports/[id]/page.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { useParams, useNavigate } from 'react-router-dom'
+import { useQuery } from '@tanstack/react-query'
+import { QueryKeys } from '@/constants/queryKeys'
+import { ReportService } from '@/features/core/services/ReportService/ReportService'
+import { format } from 'date-fns'
+
+const ReportDetailsPage: React.FC = () => {
+    const { id } = useParams()
+    const navigate = useNavigate()
+
+    const { data: report } = useQuery({
+        queryKey: [QueryKeys.SingleReport, id],
+        queryFn: async () => {
+            if (!id) return undefined
+            const reportService = new ReportService()
+            return reportService.fetchReportById(id)
+        },
+    })
+
+    if (!report) return null
+
+    const getJudgementResult = () => {
+        if (!report.adjudicatorsNullifier) return '評判中'
+        const agreeCount = report.adjudicatorsNullifier.filter(
+            (a) => a.adjudicateValue === 1,
+        ).length
+        const totalCount = report.adjudicatorsNullifier.length
+        if (totalCount === 0) return '評判中'
+        return agreeCount > totalCount / 2 ? '檢舉通過' : '檢舉不通過'
+    }
```

Comment:
I suggest to display the result by `report.status` and `report.isPassed`

Reply:
fixed

Reply:
fixed